rpm/update-spec-provides.sh

```diff
@@ -4,12 +4,27 @@
 # packaging, via the `propose-downstream` packit action.
 # The goimports don't need to be present upstream.
 
-set -eo pipefail
+set -eox pipefail
 
 PACKAGE=skopeo
 # script is run from git root directory
 SPEC_FILE=rpm/$PACKAGE.spec
 
+# Needed for golist to work
+# This is run in the packit sandbox environment for downstream tasks and
+# doesn't affect upstream repo and CI.
+GOPATHDIR=~/go/src/github.com/containers/
+mkdir -p $GOPATHDIR
+ln -sf $(pwd) $GOPATHDIR/.
+
+# Packit sandbox doesn't allow root
+# Install golist by downloading and extracting rpm
+# We could handle this in packit `sandcastle` upstream itself
+# but that depends on golist existing in epel
+# https://github.com/packit/sandcastle/pull/186
+dnf download golist
+rpm2cpio golist-*.rpm | cpio -idmv
```

Comment:
I would expect this to extract to `./usr/bin/golist`, but that’s not in `PATH`, so how does the command get executed?

Reply:
I need to switch to `./usr/bin/golist`. Thanks for pointing that out. I'll check if I can verify this in the packit local reproducible environment as well.